ui/helpers/utils/token-util.js

```diff
@@ -73,7 +73,7 @@ async function getDecimals(tokenAddress) {
     const contractMetadataInfo = getContractMetadata(tokenAddress);
 
     if (contractMetadataInfo) {
-      decimals = contractMetadataInfo.decimals;
+      decimals = contractMetadataInfo.decimals?.toString();
```

Comment:
The `?` isn't completely necessary, as all of the token objects in contract-metadata have the `decimals` attribute with a numerical value, however this is just an extra precaution should anything land there that isn't correctly formatted. We already handle `undefined` cases in `handleCustomDecimalsChange`